common/src/main/scala/explore/model/validators.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.data.NonEmptyChain
+import cats.data.NonEmptyList
+import cats.data.Validated
+import cats.syntax.all._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.string.NonEmptyString
+import explore.model.DitherNanoMeters
+import explore.model.DitherNanoMetersRange
+import explore.model.TruncatedAngle
+import explore.model.display._
+import explore.optics._
+import lucuma.core.math.Angle
+import lucuma.core.math.Axis
+import lucuma.core.math.Offset
+import lucuma.ui.optics.ValidFormatInput
+
+import scala.util.Try
+
+object validators {
+
+  // TODO TEST ALL THESE!!!!
```

Comment:
Testing will be done in upcoming PR. We need to first publish a testkit for lucuma-ui's ValidFormat.